content/pro/openbb-copilot.md

```diff
@@ -79,30 +84,14 @@ the widget has been added to the dashboard) to provide an answer.
 
 <img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/aapl_+news_chat_example.jpg" alt="aapl news openbb copilot" width="80%" height="80%" />
 
-When querying OpenBB Copilot, it is important to understand which data OpenBB
-Copilot used to answer your query. As a result, OpenBB Copilot will always cite the data
-source it referenced when responding to your query. 
-
-<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/citation_chat_example.jpg" alt="citations openbb copilot" width="80%" height="80%" />
-
 Since OpenBB Copilot can access data from any widget in the active dashboard, we
 encourage users to experiment with adding different kinds of widgets and
 experimenting with various queries.
 
 For example, OpenBB Copilot is particularly effective at summarizing earnings call transcripts
 from the "Earnings Transcripts" widget.
 
-## Creating widgets from OpenBB Copilot responses
-
-Each time OpenBB Copilot generates a response, it will have an "Create widget from text" or "Create widget from table" button. Clicking this button will create a new widget on your dashboard based on the response.
-
-| ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/text_widget_creation_button.jpg) | ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/text_widget_creation.jpg) | ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/text_widget_created.jpg) |
-|:-----------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------:|
-
-| ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/table_widget_creation_button.jpg) | ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/table_widget_creation.jpg) | ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/table_widget_created.jpg) |
-|:-----------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------:|
-
-## Querying specific widgets only
+### Explicitly as context
```

Comment:
This should be the first way to add data (since it's the easiest to understand).

Also, can I suggest:
```suggestion
### Adding widgets explicitly as context
```